UPDATING.md

```diff
@@ -316,13 +241,90 @@ The following configurations have been moved from `[core]` to the new `[logging]
 
 ### Simplification of CLI commands
```

Comment:
@ashb I not sure get your point, did you mean I reuse the exists CLI title? But I think we should put laster change in the top  of the `UPDATING.md`. Am I wrong?

Reply:
Leave it here for now, and I'll do a bigger re-ordering of all of UPDATING.md so it is a useful/easy-to-consume list for our users